util/db/dbutil_test.go

```diff
@@ -479,6 +479,16 @@ func TestReadingWhileWriting(t *testing.T) {
 
 }
 
+func TestDoubleClose(t *testing.T) {
+	partitiontest.PartitionTest(t)
+
+	acc, err := MakeAccessor("fn.db", false, true)
+	require.NoError(t, err)
+
+	acc.Close()
+	acc.Close()
```

Comment:
This should not be an acceptable workflow.
If there is a situation where this is warranted, then the caller should check it, and not the db library.